IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterwaysDomain/QuantitySets/Qto_VehicleBaseQuantities/DocQuantitySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocQuantitySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Qto_VehicleBaseQuantities" UniqueId="fde7cdb0-5c89-439d-bbf0-8c0e60700206" ApplicableType="IfcTransportElement/IfcTransportElementNonFixedTypeEnum(.VEHICLE.),IfcTransportElement/IfcTransportElementNonFixedTypeEnum(.VEHICLETRACKED.),IfcTransportElement/IfcTransportElementNonFixedTypeEnum(.VEHICLEMARINE.),IfcTransportElement/IfcTransportElementNonFixedTypeEnum(.VEHICLEAIR.),IfcTransportElement/IfcTransportElementNonFixedTypeEnum(.ROLLINGSTOCK.),IfcTransportElementType/IfcTransportElementNonFixedTypeEnum(.VEHICLE.),IfcTransportElementType/IfcTransportElementNonFixedTypeEnum(.VEHICLETRACKED.),IfcTransportElementType/IfcTransportElementNonFixedTypeEnum(.VEHICLEMARINE.),IfcTransportElementType/IfcTransportElementNonFixedTypeEnum(.VEHICLEAIR.),IfcTransportElementType/IfcTransportElementNonFixedTypeEnum(.ROLLINGSTOCK.)">
+<DocQuantitySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Qto_VehicleBaseQuantities" UniqueId="fde7cdb0-5c89-439d-bbf0-8c0e60700206" ApplicableType="IfcVehicle/ROLLINGSTOCK,IfcVehicle/VEHICLEAIR,IfcVehicle/VEHICLEMARINE,IfcVehicle/VEHICLETRACKED,IfcVehicle/VEHICLE">
```

Comment:
Couldn't applicability be IfcVehicle (and IfcVehicleType)?

Reply:
The (IFC4 I think) agreement is that applicability is documented only on occurrence entities. With general usage in place, it is clear which `IfcTypeObject` can be associated to the IfcObject.